src/api/actions/auth/authActions.ts

```diff
@@ -0,0 +1,13 @@
+import { Action } from '../../types';
+import { LoginPayload } from './auth.types';
+
+export const loginAction = (values: LoginPayload): Action => ({
+  method: 'POST',
+  url: '/auth',
+  data: values,
+});
+
+export const checkIsLoggedInAction = (): Action => ({
```

Comment:
Tu i powyżej - jako że to jest funkcja tworząca akcję, to powinna się zaczynać od `create`, a kończyć na `Action`, więc `createCheckIsLoggedInAction`